src/controller/CHIPDeviceController.cpp

```diff
@@ -220,23 +218,26 @@ void DeviceController::OnLocalNOCChainGeneration(void * context, CHIP_ERROR stat
 
     DeviceController * controller = static_cast<DeviceController *>(context);
 
-    Transport::FabricInfo * const fabric = controller->mFabrics.FindFabricWithIndex(controller->mFabricIndex);
+    Transport::FabricInfo newFabric;
```

Comment:
Is there a better namespace than "Transport" for FabricInfo? It's a core system model concept, not just transport related.

Reply:
We can move it to another namespace, given SecureSessionMgr is mostly independent of it now.

Reply:
Filed this issue to track it: https://github.com/project-chip/connectedhomeip/issues/8911